<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuUI</class>
 <widget class="QDialog" name="MenuUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>386</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Hargoile Menu</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="routeUploadBtn">
     <property name="text">
      <string>Upload Raw Route</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="reducedRouteUploadBtn">
     <property name="text">
      <string>Upload Compressed Route</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="testRouteUploadBtn">
     <property name="text">
      <string>Upload Compressed Route [Test]</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="newRouteBtn">
     <property name="text">
      <string>Record New Route</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="unlinkBtn">
     <property name="text">
      <string>Unlink Account</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="quitBtn">
     <property name="text">
      <string>Quit Hargoile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="menuButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
